.github/workflows/services_build_deploy_dev.yml

```diff
@@ -99,6 +102,14 @@ jobs:
           GH_TOKEN: ${{ secrets.SHIELDS_GH_TOKEN }}
           REPO_NAME: ${{ github.repository }}
 
+      - name: Create .env File if Exists
+        run: |
+          if [ -f ".env.$APP_ENV" ]; then
+            mv -f ".env.$APP_ENV" .env
+            rm -f .env.*
+            cat .env
```

Comment:
Should we expect .env files shared on github repositories?

Reply:
I see now the connection of this changes with this PR: https://github.com/Cell-Ed/cei-svc-chatbot/pull/10.
I don't think this is the best pattern to shared environment variables in our secrets. 
I suggest to leverage the secrets pattern that it's already in use. I'll provide more details on https://github.com/Cell-Ed/cei-svc-chatbot/pull/10

Reply:
Sorry for the inconvenience occurred with lack of description the [LMSNEXT-530](https://celled.atlassian.net/browse/LMSNEXT-530). 

Expecting to do all the required changes to deploy chatbot API into the staging environment under same jira.

Reply:
Review comment is incorporated with the discussion we had. Thank you very much for your inputs and suggested [hybrid approach](https://celled.atlassian.net/wiki/x/AwDOMw) found with the hyperlink